README.md

```diff
@@ -1,5 +1,9 @@
 # Zeppelin Converter
 
+### Requirements
+
+- Python 3.\*
+- Cairo: `brew install cairo`
```

Comment:
This works only on OSX. Instead of giving the exact command, you could explain what Cairo is and why it's needed and suggest to install it using watchever package system is available on the user machine.